cfme/tests/infrastructure/test_config_management.py

```diff
@@ -144,9 +146,33 @@ def test_config_system_tag(request, config_system, tag):
     """
     config_system.add_tag(tag=tag, details=False)
     tags = config_system.get_tags()
-    assert '{}: {}'.format(tag.category.display_name, tag.display_name) in \
-        ['{}: {}'.format(t.category.display_name, t.display_name) for t in tags], \
-        "Failed to setup a configuration system's tag"
+    msg = "Failed to setup a configuration system's tag"
+    tag_info = '{}: {}'.format(tag.category.display_name, tag.display_name)
+    assert tag_info in ['{}: {}'.format(t.category.display_name, t.display_name) for t in tags], msg
+
+
+@pytest.mark.tier(3)
+@test_requirements.tag
+@pytest.mark.uncollectif(lambda config_manager_obj: config_manager_obj.type != "Ansible Tower")
+def test_ansible_tower_job_templates_tag(request, config_manager, tag):
+    """
+    Polarion:
+        assignee: anikifor
+        initialEstimate: 1/4h
+        casecomponent: Ansible
+        caseimportance: high
+
+    Bugzilla:
+        1673104
+    """
+    collection = config_manager.appliance.collections.ansible_tower_job_templates
+    job_template = collection.all()[0]
```

Comment:
optional:
I would try to catch random template form job_templates if not available then either `create` or `skip` template with a proper message. It will help to avoid IndexError if not template available for testing.